lib/rake/build.rake

```diff
@@ -10,6 +10,9 @@ namespace :build do
       ChatClient.log 'Installing <b>apps</b> dependencies...'
       RakeUtils.npm_install
 
+      # Workaround for https://github.com/sass/node-sass/issues/1804
+      RakeUtils.npm_rebuild 'node-sass'
```

Comment:
About how long does this workaround-step take to run? This runs on every single build, which could add a significant amount of additional time to every single step of our pipeline (most significantly launching new frontend instances, which we want to keep as short as possible for auto-scaling purposes). If it's < 1 minute it should be no problem, if it's > 1 minute let's try and find a smarter way of running this more selectively, as long as this workaround will be necessary moving forward.